### PR TITLE
Firecert 1837 Create new handler in FCA to make lifecycle calls

### PR DESCRIPTION
**Pull request by neeradanelxsi**
# Description
Enter a detailed description of the PR here, including the nature and purpose of the changes.

## JIRA Ticket
Include a link to the JIRA Ticket linked to this PR.
[Issue-0000](https://your-jira-domain.com/browse/Issue-0000)

## Additional Info (Test steps / Steps to reproduce etc.)
Details on how to test the feature

### CODE REVIEW

File: src/pubsub/handlers/LifecycleMethodHandler.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export default class LifecycleMethodHandler extends BaseHandler {
+  constructor(handlerName) {
+    super(handlerName);
+  }
+  async handle(message) {
+    const sdkInvokerInfo = new Test_Runner();
+    const lifecycleApiResponse = await sdkInvokerInfo.invokeLifecycleAPI(message.params);
+    return lifecycleApiResponse;
+  }
```
Comment (neeradanelxsi) [inline src/pubsub/handlers/LifecycleMethodHandler.js:34]: we need to have error handling for this also

File: test/unit/IntentReader.test.js [hunk 2]
```diff
@@ -64,6 +64,31 @@ jest.mock('@firebolt-js/sdk/dist/lib/Transport/index.mjs', () => {
...
+      finish: () => {},
+    },
+    Localization: {},
+    Metrics: {},
+    Profile: {},
+    Parameters: {},
+    SecondScreen: {},
+  };
+});
+
```
Comment (neeradanelxsi) [inline test/unit/IntentReader.test.js:91]: what is this mock for ? its not used anywhere . there are linting issues too

Comment (anjalimukundan) [reply test/unit/IntentReader.test.js:91]: unit testcases were failing without this mock after adding the new handler.

File: src/pubsub/handlers/LifecycleMethodHandler.js (hunk 1 above)
Comment (anjalimukundan) [reply src/pubsub/handlers/LifecycleMethodHandler.js:34]: added as a try catch block

File: test/unit/IntentReader.test.js (hunk 2 above)
Comment (anjalimukundan) [reply test/unit/IntentReader.test.js:91]: linting issues are fixed

**Review by kevinshahfws — APPROVED** [review]

**Pull request merged**